content/docs/04.5-devx/05-app-profile.md

```diff
@@ -18,21 +18,105 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 App Profiles are templates created with preconfigured services required for Palette Virtual Cluster deployment. App Profiles provide a way to drive consistency across virtual clusters. 
 
-You can create as many app profiles as required. An App Profile can be created to meet specific types of workloads on Palette Virtual Clusters. For example, you may create a development app profile with Helm Charts, Manifest, different database applications, etc.
+You create App Profiles to meet specific types of workloads on your Palette Virtual Clusters. For example, you may want to create a development App Profile with Helm Charts, Manifests, and different database applications. You can create as many App Profiles as you need.
 
-To learn more about App Profiles and its capabilities checkout the following resources.
+Several Services are available to build App Profiles.
 
-<br />
+## Services
```

Comment:
```suggestion
# Services
```
I think we should make this a higher level. That way, the services below, such as databases, fall under this umbrella.